chore: Add Naming Series for Project DocType

michellealva opened the pull request:
- Added Naming Series (PROJ.####) for Project DocType.

**Before:**

![image](https://user-images.githubusercontent.com/50285544/91661218-3c23f580-eaf8-11ea-9ba7-008c3033716b.png)


**After:**

![image](https://user-images.githubusercontent.com/50285544/91661235-5362e300-eaf8-11ea-813d-b47d0a2ceedd.png)

michellealva committed:
feat: Add Naming Series for Project DocType

michellealva commented:
@Mergifyio backport version-12-hotfix

michellealva committed:
fix: Change property of field

marination committed:
Merge branch 'develop' into naming-series-proj

marination committed:
fix: Crop Cycle Test

marination committed:
fix: Dont Copy or Print Naming Series

hrwX commented on erpnext/projects/doctype/project/project.json:
```diff
@@ -440,13 +441,24 @@
+   "fieldtype": "Select",
+   "label": "Series",
+   "no_copy": 1,
+   "options": "PROJ.####",
```
```suggestion
   "options": "PROJ-.####",
```

michellealva committed:
fix: Change naming series

Co-authored-by: Himanshu <himanshuwarekar@yahoo.com>

gavindsouza committed:
Merge branch 'develop' into naming-series-proj

nabinhait committed:
Merge branch 'develop' into naming-series-proj

marination committed:
Merge branch 'develop' into naming-series-proj

marination committed:
chore: Fix Tests, use project name instead of project_name field

marination committed:
Merge branch 'develop' into naming-series-proj

nabinhait committed:
Merge branch 'develop' into naming-series-proj

marination committed:
Merge branch 'develop' into naming-series-proj

marination committed:
Merge branch 'develop' into naming-series-proj

marination committed:
fix: Clear merge conflicts

- 'get_project_template' was removed a month back
- in 'make_project' insert the project directly as a check for it is there in the beginning.

marination committed:
Merge branch 'develop' into naming-series-proj
